tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "strict": false,
     "noUnusedLocals": false,
     "lib": ["ES7"],
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "skipLibCheck": true
```

Comment:
What's the reason for adding in the `skipLibCheck`?

Reply:
I pushed that by mistake as something didn't work for me as I don't quite understand how all of libs are connected and linked to each other. I will remove that in next commit with fix tests